networking/ingress-controller-dnsmgt.adoc

```diff
@@ -0,0 +1,37 @@
+:_content-type: ASSEMBLY
+[id="ingress-controller-dnsmgt"]
+= Configuring an Ingress Controller for disabling DNS Management
+include::_attributes/common-attributes.adoc[]
+:context: ingress-controller-dnsmgt
+
+toc::[]
+
+As a cluster administrator, when you create an Ingress Controller, the Operator manages the DNS records by default. This has some limitations when the required DNS zone is different from the cluster DNS zone or when DNS zone is hosted outside the cloud provider. In such scenarios, the Operator reports a degraded state after attempting to provision the required resources in the cloud provider because the wildcard DNS record can not be created on the cloud provider.
+
+Day 2 operations add support to the Operator to have the option to manage DNS record lifecycle on the cloud provider. As a cluster administrator, you can create or modify an Ingress Controller with `dnsManagementPolicy` to specify whether the Operator needs to manage the DNS record lifecycle or not.
```

Comment:
```suggestion
As a cluster administrator, you can create or modify an Ingress Controller with `dnsManagementPolicy` to specify whether the Operator needs to manage the DNS record lifecycle or not.
```

Reply:
nit: This seems redundant in my opinion.

Reply:
Removed